### PR TITLE
fix pose_to_segments elan process

### PR DESCRIPTION
**Pull request by J22Melody**

### CODE REVIEW

File: pose_to_segments/bin.py [hunk 1]
```diff
@@ -62,26 +63,30 @@ def get_args():
     parser.add_argument('-i', required=True, type=str, help='path to input pose file')
     parser.add_argument('-o', required=True, type=str, help='path to output elan file')
     parser.add_argument('--video', default=None, required=False, type=str, help='path to video file')
+    parser.add_argument('--subtitle', default=None, required=False, type=str, help='path to subtitle file')
```
Comment (AmitMY) [inline pose_to_segments/bin.py:66]: `subtitle` should be `subtitles` (same below)
(a subtitle refers to a single cue)

File: pose_to_segments/bin.py [hunk 2]
```diff
@@ -102,7 +107,18 @@ def main():
     for tier_id, segments in tiers.items():
         eaf.add_tier(tier_id)
         for segment in segments:
-            eaf.add_annotation(tier_id, segment["start"] * fps, segment["end"] * fps)
+            eaf.add_annotation(tier_id, int(segment["start"] / fps * 1000), int(segment["end"] / fps * 1000))
+
+    if args.subtitle:
+        eaf.add_tier("SUBTITLE")
```
Comment (AmitMY) [inline pose_to_segments/bin.py:113]: `import srt` locally here, not globally to make it an optional dependency

File: pose_to_segments/bin.py [hunk 3]
```diff
@@ -62,26 +63,30 @@ def get_args():
...
-    model = load_model(os.path.join(install_dir, "dist", "model_E1s-1.pth"))
+    model = load_model(os.path.join(install_dir, "dist", args.model))
 
     print('Estimating segments ...')
     probs = predict(model, pose)
 
-    sign_segments = probs_to_segments(probs["sign"])
-    sentence_segments = probs_to_segments(probs["sentence"], .8, .8)
+    sign_segments = probs_to_segments(probs["sign"], 60, 50)
+    sentence_segments = probs_to_segments(probs["sentence"], 85, 80)
```
Comment (AmitMY) [inline pose_to_segments/bin.py:89]: paper says 90, 90 for this step

File: pose_to_segments/bin.py [hunk 4]
```diff
@@ -62,26 +63,30 @@ def get_args():
...
     return parser.parse_args()
 
-
 def main():
     args = get_args()
 
     print('Loading pose ...')
     with open(args.i, "rb") as f:
-        pose = process_pose(Pose.read(f.read()))
-
```
Comment (AmitMY) [inline pose_to_segments/bin.py:75]: maybe 
```
pose = Pose.read(f.read())

if 'E4' in args.model:
  pose = process_pose(pose, optical_flow=True, hand_normalization=True)
else:
  pose = process_pose(pose)

**Review by AmitMY — CHANGES_REQUESTED** [review]
thanks for doing this work :)

File: pose_to_segments/bin.py (hunk 3 above)
Comment (J22Melody) [reply pose_to_segments/bin.py]: Sure, let's also use this on SignTube then, I set `85, 80` to align with SignTube.

**Review by AmitMY — APPROVED** [review]

**Pull request merged**